src/main/java/com/fryrank/dal/ReviewDALImpl.java

```diff
@@ -38,48 +40,50 @@ public class ReviewDALImpl implements ReviewDAL {
     @Autowired
     private MongoTemplate mongoTemplate;
 
+    private static final List<AggregationOperation> AGGREGATION_OPERATIONS_FOR_PUBLIC_UISER_METADATA_COLLECTION_JOIN =
```

Comment:
nit: spelling (`UISER`)

Reply:
Oops, fixed in [446c814](https://github.com/NickPriv/FryRankBackend/pull/87/commits/446c81426cb7e82fa809735a2120f0b90cbfa782).